### DISCUSSION TITLE
Coupled model tuning: what can land do (or what have we done)?

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/LMWG_dev#75] [Q&A]
A few issues with the latest coupled model runs, that have been discussed at the [CESM project meeting](https://docs.google.com/presentation/d/1nOlCXCv37-_SSE-ARAloq9OrY7OqQNOU/edit?usp=sharing&ouid=110501992574177180138&rtpof=true&sd=true) and [CAM7 meeting](https://docs.google.com/presentation/d/1-G7HRsMXC4DRH6GDuJuYyi6PuiB7Y3aT8xzsFQ1wCMY/edit?usp=sharing). 
 Trying to take some notes here and start a discussion to see if the land model may have any role to play here?  Notably,  
- **high RESTOM**
- **low precip biases over the Amazon** (compared to land only CRU-JRA runs, below). Note also high biases in tropical Africa and Asia.
  - precip biases are also associated with [low humidity](https://github.com/user-attachments/assets/a3dc0a4a-0056-44ce-afae-50b8367eaaa5), [low LAI](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_TLAI.gif), and [high incoming SW](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_FSDS.gif) radiation over the region.
- **High incoming solar (and presumably low clouds) over northern hemisphere mid-latitudes**.
<p align="center"> <img src="https://github.com/user-attachments/assets/eb989e35-6880-4a74-93bf-f73016bc0b77" width="300" /></p>

*At this point it's not clear if the high RESTOM in [cesm_dev 116](https://github.com/NCAR/cesm_dev/issues/19) could be connected to these regional biases in the Amazon.  It's also not clear if land model parameterizations have caused, or could help address, either RESTOM or climatological issues?*. **NOTE:** high restom seems like a persistent [feature of #116 that's better in subsequent B-cases](https://docs.google.com/presentation/d/11AfW5dmjzlxPFnVsHu4OblCXIy7kgrHRG8ZEGxgF8jA/edit?usp=sharing)

We do have some suggestions from [recent work](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2024GL108372) by @czarakas and friends suggests that land model parameters can influence regional precipitation.  How much can land model parameters help bring back rains (and clouds) over the Amazon, or reduce incoming SW that hits the land in mid-latitudes?

**Amazon biases were investigated with F-cases, see #76 and #77.**

**Mid-latitude clouds / incoming solar was investigated with #78-81**

----
If you're still interested here are some additional places to look at LMWG diagnostics
**Relevant Land Diagnostics**
 [B-116 vs. I-5.3](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/setsIndex.html)

[B-116 vs. B-112](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20/setsIndex.html)

[B-112 vs. B-111](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/setsIndex.html)

[B-116 vs. F-0008](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_14_23-f.e30_alpha04a.FMTHIST.ne30_L93_tuning.008_2000_2009/setsIndex.html), the F-case is a CLM6-SP case

Lots of [LMWG diagnostics posted](https://webext.cgd.ucar.edu/BLT1850/) from B cases here (thanks @olyson)

--------------
### Notes on the dry amazon bias
- We are unable to replicate the bias in F-cases [#77 vs. #76 diags](https://webext.cgd.ucar.edu/F2000climo/f.e30_alpha04a.FMT2000BgcCrop.52params.lmwg_dev_77/lnd/f.e30_alpha04a.FMT2000BgcCrop.52params.lmwg_dev_77.11-20-f.e30_alpha04a.FMT2000BgcCrop.CTRL.lmwg_dev_76.11-20/setsIndex.html)
  - Increasing tropic LAI in F-cases also does not increase precip over the Amazon
- Initializing B cases updated initial conditions does not solve the issue, LAI declines sharply [B-120 vs. B-118 diags](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.120/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_7_16-b.e30_beta04.BLT1850.ne30_t232_wgx3.118_11_20/setsIndex.html)
- Hot, dry conditions in the Amazon came in with [B-112 vs. B-104 diags](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.104_11_20/setsIndex.html), declines in LAI but without decreases in precip.

@adamrher, @dlawrenncar, @linniahawkins, @djk2120, @katiedagon, @aswann, @czarakas,  @olyson, @slevis-lmwg

**Comment by wwieder** [comment]
@czarakas & @aswann can you get a sense of the strength of parameters in controlling the EOF regionally from your PPE work, specifically in the tropics?

**Comment by wwieder** [comment]
Potentially useful here are the regional climatology here for the 
### Amazon in an B_beta04_clm6BGC (red) and F_beta04_clm6SP (blue) case

![image](https://github.com/user-attachments/assets/7b498e84-3dd5-4af4-b15e-55cfae486c69)

### And for the same B_beta04_clm6BGC (red) with an I_clm6BGC (blue) case

![image](https://github.com/user-attachments/assets/0aa13cea-e8f2-47f4-9cfc-de30bdf2afce)

@dlawrenncar, I guess we'll have to se if an F_beta04_clm6BGC case replicated the behaviors that we're seeing in the B-case?

**Comment by aswann** [comment]
<img width="1174" alt="Screenshot 2024-11-26 at 5 59 51 PM" src="https://github.com/user-attachments/assets/d6c3b251-7cd2-479d-8134-d5b2138062b8">
<img width="1173" alt="Screenshot 2024-11-26 at 5 59 41 PM" src="https://github.com/user-attachments/assets/f4b8ac0f-61a9-416a-b2d2-d1de196f3107">
<img width="1177" alt="Screenshot 2024-11-26 at 5 59 27 PM" src="https://github.com/user-attachments/assets/ce28d03f-5823-440a-bb67-81b31f41c0cd">
Here is one map of change in precip for each ensemble member in the CESM2 PPE that Claire ran. The file was too big to upload so I cut it into three pieces

**Reply by linniahawkins** [reply]
Thanks @aswann is there a way to tell which is the max/min setting for each parameter?

**Reply by aswann** [reply]
That would be helpful wouldn't it! That was the plot I had on hand. I re-made just the kmax maps. 0026 is kmax min, and 0027 is kmax max

![RAIN_globalmaps_maxleafwetfrac_-2](https://github.com/user-attachments/assets/b7a88368-7349-4d3e-a7ac-f4ef8248bbef)
![RAIN_zonalprofiles_maxleafwetfrac](https://github.com/user-attachments/assets/abd97ca8-aaeb-43e3-8f20-f39d3520b6b4)

**Reply by czarakas** [reply]
The min and max values vary by PFT - the paramfiles are OAAT0159.nc and OAAT0148.nc [here](https://github.com/czarakas/coupled_PPE/tree/main/data/paramfiles) or in `/glade/u/home/czarakas/coupled_PPE/data/paramfiles/`
the min values range from 2.3e-10 to 1.5e-08 (depending on the PFT), and the max values range from 1.294e-09 to 4.0278e-08

**Reply by linniahawkins** [reply]
Thanks Claire and Abby - this is really helpful!

**Reply by wwieder** [reply]
@czarakas and @aswann are these runs on campaign store anywhere?  We're interested in looking at clouds and ET from these runs. @linniahawkins, @dlawrenncar

**Reply by aswann** [reply]
I went ahead and plotted total ET and low clouds for kmax. I also realized that my unit was wrong on the RAIN plot, so updated here. 0026 is kmax min, and 0027 is kmax max
![CLDLOW_globalmaps_maxleafwetfrac_](https://github.com/user-attachments/assets/19ddc8e6-d992-47ac-891e-741b421acc38)
![CLDLOW_zonalprofiles_maxleafwetfrac](https://github.com/user-attachments/assets/01f821fb-7a91-4309-9161-a37915a466eb)
![EFLX_LH_TOT_globalmaps_maxleafwetfrac_](https://github.com/user-attachments/assets/f7469b3d-c04f-4d23-8b3b-8269fb2d4d7a)
![EFLX_LH_TOT_zonalprofiles_maxleafwetfrac](https://github.com/user-attachments/assets/1294f180-78bc-4242-89ad-7d559cd9baa8)
![RAIN_globalmaps_maxleafwetfrac_-3](https://github.com/user-attachments/assets/e416f2d1-f215-4123-875c-8d804a72d56c)
![RAIN_zonalprofiles_maxleafwetfrac-2](https://github.com/user-attachments/assets/347c5b00-92b4-4f9d-b3dc-0a17111ad48a)

**Reply by aswann** [reply]
If you are just interested in how clouds respond to ET in general, this is the shortwave cloud feedback per change in ET across the whole CESM2 PPE. This tells where you are likely to get short wave clouds forcing from changes in land overall. I'll also make the maps of changes in FSDS for every ensemble member.
<img width="1815" alt="SWcloud_feedbacks_CESM" src="https://github.com/user-attachments/assets/524a967d-5f6f-4a1b-98d4-b050330f8912">

**Reply by aswann** [reply]
Here are snapshots of change in FSDS for all ensemble members
<img width="1729" alt="Screenshot 2024-12-05 at 4 01 49 PM" src="https://github.com/user-attachments/assets/d3458a5e-35bf-4952-b909-ca048d9a4048">
<img width="1727" alt="Screenshot 2024-12-05 at 4 01 35 PM" src="https://github.com/user-attachments/assets/0a6e0374-67f0-42bd-8cdb-f99e6d194f06">

**Reply by wwieder** [reply]
Hit those Luna parameters to influence clouds! Am I seeing this right?

**Reply by wwieder** [reply]
It's also interesting how divergent the mid-latitude changes are compared to what's happening over the arctic (e.g. d_max and maximum leaf wetted fraction) where we seemingly can decrease FSDS over land and increase it over Arctic ocean.

**Reply by wwieder** [reply]
quick question, @aswann how are these plots arranged with left and right plots for low and high parameter ranges, respectively?

Specifically we're interested in results from d_max and uplim_desdruct_metamorph that decrease FSDS over mid-latitude land

**Comment by aswann** [comment]
Yes! If you want to dive right in the output is here: `/glade/campaign/cgd/tss/people/czarakas/CoupledPPE/coupled_simulations`

It would probably be useful if @czarakas or I could provide you with some scripts to get started but I don't have those prepared. At a minimum you can find which run is which with this table:
`/glade/u/home/czarakas/coupled_PPE/code/02_set_up_ensemble/CLM5PPE_coupledPPE_crosswalk.csv`

I could probably make some more plots of other variables like the ones of RAIN for kmax above without too much trouble if you let me know what you want to look at.

**Comment by dlawrenncar** [comment]
Interesting.  Or nstem seems like another candidate.  I wonder if changing
nstem would have less of an impact on GPP, so more focused on the water /
energy budget?

**Comment by olyson** [comment]
b.e30_beta04.BLT1850.ne30_t232_wgx3.120 (https://github.com/NCAR/cesm_dev/issues/25) had run 16 years by last night so I ran lnd diagnostics compared to b.e30_beta04.BLT1850.ne30_t232_wgx3.118 (https://github.com/NCAR/cesm_dev/issues/23) .  Thus showing the effect of land initial conditions:

https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.120/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_7_16-b.e30_beta04.BLT1850.ne30_t232_wgx3.118_11_20/setsIndex.html

It looks like TLAI in the Amazon starts off with values of 4-5 in the first year but then dies off to the level we see in 118.  Here is the annual mean for the first year of 120 (ncview /glade/derecho/scratch/oleson/ANALYSIS/climo/b.e30_beta04.BLT1850.ne30_t232_wgx3.120_7_16/FV_192x288_b.e30_beta04.BLT1850.ne30_t232_wgx3.120_7_16_ANN_ALL.nc)

![TLAI_Legend](https://github.com/user-attachments/assets/fb389fd9-8e10-4b96-b2e9-5b6966fca1d5)

![TLAI](https://github.com/user-attachments/assets/782731bb-c152-41e2-86aa-0c8d37227a75)

And the annual mean trend in the Amazon:


![TLAI_Annual_Mean_Trend](https://github.com/user-attachments/assets/92acd921-8f59-4985-aa81-a9e5cc43069e)

**Comment by linniahawkins** [comment]
Yikes - so if it's not an initial conditions problem, and this same parameterization of tropical trees gives use LAI's of >10 in F-cases and I-Cases, can we attribute it to the dry bias in CESM and the high water use I've prescribed for tropical trees? 

initial ideas:
1) revert to ctsm5.2 parameters for BETT (very high LAI bias in I and F cases) and see if those survive in a B-case with updated initial conditions. 
2) Try reducing tropical LAI through leafcn & slatop only (which will result in lower ET and less water stress).

**Reply by aswann** [reply]
We saw similar LAI die-off behavior in the Amazon when we made a really extreme experiment where the plants thought they always had very small VPD. (We set VPD to very small inside the Medlyn conductance equation only). What's the humidity in the data atmosphere vs CAM?

**Reply by olyson** [reply]
Slide 3 of Will's presentation yesterday showed the difference in RH2M between 116 (fully-coupled) and ctsm5.3 driven by CRUJRA.  
Also shown here:

https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_RH2M.gif

**Reply by olyson** [reply]
Can also be seen in the forcing specific humidity (QBOT):

https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_10_19-ctsm53n04ctsm52028_f09_pSASU_181_200/set2/set2_ANN_QBOT.gif

**Comment by wwieder** [comment]
Thanks for posting these, @olyson.  I don't think this is an issue we need to fix, @linniahawkins, as it seems more like an issue with how much moisture is coming into the Amazon that's keeping it too hot and dry.  What do you think @dlawrenncar ?
![image](https://github.com/user-attachments/assets/e31334eb-2741-4484-a3bc-b9ef51bbc4c0)

**Comment by dlawrenncar** [comment]
@olyson @wwieder It does seem like this is most likely a coupled model problem.  To the extent that we have the right diagnostics, can you trace back to which coupled model tag / simulation this dry bias starts to show up?  I'm guessing that it will be with the new beta04 tag, but we should confirm.  If beta04, then we probably need to have a conversation as a CESM group about what could be the source (especially since this is not apparent in F-cases).  It does seem pretty unlikely that land parameters or processes are playing a dominant role, especially since we cannot reproduce the problem in the F cases.

**Reply by olyson** [reply]
It looks like we started losing Amazon (and more broadly the tropics and boreal forest) LAI in b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112.  It looked better in b.e30_beta02.BLT1850.ne30_t232.111.  Note that both of these are compared to 104.  So, alpha03d versus beta02.  But QBOT and PREC between 112 and 111 don't look that different to me.  I will run the diagnostics to compare these two runs.

![set2_ANN_TLAI_112](https://github.com/user-attachments/assets/3257d3a3-2f5d-4afa-b933-4d5ca82a3db2)


![set2_ANN_TLAI_111](https://github.com/user-attachments/assets/99a9b501-f8b4-4956-bddc-d0f61a32274b)

**Reply by wwieder** [reply]
Thanks Keith, it looks like the ocean changes that got rid of the double ITCZ also made a hit on Amazon LAI.  Interestingly it's not changes in precip, but warmer temperature and declines in humidity that made the [hit in LAI in #112, relative to #104](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.104_11_20/setsIndex.html)

**Reply by olyson** [reply]
Diagnostics for 112 compared to 111 [here](https://webext.cgd.ucar.edu/BLT1850/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112/lnd/b.e30_alpha03d.BLT1850.ne30_t232_wgx3.112_11_20-b.e30_beta02.BLT1850.ne30_t232.111_11_20/setsIndex.html)

**Comment by wwieder** [comment]
I'll note that between 111 and 112 we also move from CTSM5.2 datasets/parameters to 5.3, with new parameterization for tropical forests and a new fire model!  

I've also started a focused discussion for tropical LAI and mid-latitude shortwave biases on the CESM_dev discussions.
